protocol/0028-GOVE-governance.md

```diff
@@ -453,11 +444,9 @@ Below `*` stands for any of `asset, market, updateMarket, updateNetParam, freeFo
 
 
 ### Freeform governance proposals
-- [ ] A freeform governance proposal with a description field that is empty, or not between 0 and 255 characters, will be rejected (<a name="0028-GOVE-019" href="#0028-GOVE-019">0028-GOVE-019</a>)
-- [ ] A freeform governance must contain a hash field and it must not be null, but no other check is done to verify it (<a name="0028-GOVE-020" href="#0028-GOVE-020">0028-GOVE-020</a>)
-- [ ] A freeform governance must contain a link field and it must not be null, but no other check is done to verify it (<a name="0028-GOVE-021" href="#0028-GOVE-021">0028-GOVE-021</a>)
+- [ ] A freeform governance proposal with a description field that is empty, or not between 0 and 10,000 characters, will be rejected (<a name="0028-GOVE-019" href="#0028-GOVE-019">0028-GOVE-019</a>)
```

Comment:
As above, should the min. length be 1, or more if it is populated?

Reply:
done here: https://github.com/vegaprotocol/specs/pull/1186/commits/0a9f0f2a1ce1de3301dafba2bd1496fa38166de2